### PR TITLE
docs: fix missing app -> service renames

### PR DESCRIPTION
**Pull request by watson**
Fixes elastic/apm-agent-nodejs#237

### CODE REVIEW

File: docs/guide/index.asciidoc [hunk 1]
```diff
@@ -220,8 +220,8 @@ Then configure the elastic-apm-node module inside your service by adding the fol
 ----------------------------------
 // Add this to the VERY top of the first file loaded in your app
```
Comment (simitt) [inline docs/guide/index.asciidoc]: you could also rename `app` to `service` here, but not mandatory..

**Comment by beniwohli** [comment]
@watson would it maybe make sense to not include a configuration snippet and only link to the agent docs? The more places we have this snippet, the more we have to remember to update it next time.

**Comment by watson** [comment]
@beniwohli Absolutely. I never really questioned having it on this page as it was there from the beginning. But I'll revisit this later 👍

File: docs/guide/index.asciidoc (hunk 1 above)
Comment (watson) [reply docs/guide/index.asciidoc]: From a Node.js developers perspective, I think it's more clear to use "app" in this location. But I'm happy to reevaluate if you feel it's confusing

**Pull request merged**